daemon/src/rollover_taker.rs

```diff
@@ -348,16 +383,24 @@ pub struct RollOverRejected;
 
 /// Message sent from the spawned task to `rollover_taker::Actor` to
 /// notify that rollover has finished successfully.
-pub struct RolloverSucceeded {
+struct RolloverSucceeded {
     dlc: Dlc,
 }
 
 /// Message sent from the spawned task to `rollover_taker::Actor` to
 /// notify that rollover has failed.
-pub struct RolloverFailed {
+struct RolloverFailed {
     error: anyhow::Error,
 }
 
+/// Message sent from the spawned task to `rollover_taker::Actor` to
+/// notify that the timeout has been reached.
+///
+/// It is up to the actor to reason whether or not the protocol has progressed since then.
+struct MakerResponseTimeoutReached {
+    timeout: Duration,
+}
```

Comment:
This change is likely among the best to pick out of this branch. Not sure if many of the others are worth it!